src/cov/Coverage.java

```diff
@@ -519,8 +519,13 @@ public static void initCoverage(Class<?> cls) {
    */
   public static Member getMemberContaining(CoverageAtom cov) {
     if (cov == null) throw new IllegalArgumentException("cov cannot be null.");
-    Class<?> cls = Reflection.classForName(cov.getClassName());
-    initCoverage(cls);
+
+    try {
+      initCoverage(Class.forName(cov.getClassName()));
+    } catch (ClassNotFoundException e) {
+      //ignor
```

Comment:
Under what circumstances is CoverageAtom not a class name?  If that situation is an error, then I feel strongly that Randoop should throw an exception rather than silently masking the error.
Also, when I asked the circumstances, I meant to document them in the code rather than just in the code review conversation, but that might not have been clear.